scripts/sct_straighten_spinalcord.py

```diff
@@ -15,235 +15,242 @@
+        default = 'nurbs')
+    optional.add_argument(
+        '-centerline-smooth',
+        metavar=Metavar.int,
+        type=int,
+        help='Degree of smoothing for centerline fitting. Only use with -centerline-algo {bspline, linear}.',
+        default=10)
 
-    parser.add_option(name='-centerline-algo',
-                      type_value='multiple_choice',
-                      description='Algorithm for centerline fitting.',
-                      mandatory=False,
-                      example=['bspline', 'linear', 'nurbs'],
-                      default_value='nurbs')
-    parser.add_option(name='-centerline-smooth',
-                      type_value='int',
-                      description='Degree of smoothing for centerline fitting. Only use with -centerline-algo {bspline, linear}.',
-                      mandatory=False,
-                      default_value=10)
+    optional.add_argument(
+        "-param",
+        metavar=Metavar.list,
+        help="R|Parameters for spinal cord straightening. Separate arguments with ','."
+             "\nprecision: [1.0,inf[. Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with hanning fitting method. Default=2"
+             "\nthreshold_distance: [0.0,inf[. Threshold at which voxels are not considered into displacement. Increase this threshold if the image is blackout around the spinal cord too much. Default=10"
+             "\naccuracy_results: {0, 1} Disable/Enable computation of accuracy results after straightening. Default=0"
+             "\ntemplate_orientation: {0, 1} Disable/Enable orientation of the straight image to be the same as the template. Default=0",
+        required=False)
 
-    parser.add_option(name="-param",
-                      type_value=[[','], 'str'],
-                      description="Parameters for spinal cord straightening. Separate arguments with ','."
-                                  "\nprecision: [1.0,inf[. Precision factor of straightening, related to the number of slices. Increasing this parameter increases the precision along with increased computational time. Not taken into account with hanning fitting method. Default=2"
-                                  "\nthreshold_distance: [0.0,inf[. Threshold at which voxels are not considered into displacement. Increase this threshold if the image is blackout around the spinal cord too much. Default=10"
-                                  "\naccuracy_results: {0, 1} Disable/Enable computation of accuracy results after straightening. Default=0"
-                                  "\ntemplate_orientation: {0, 1} Disable/Enable orientation of the straight image to be the same as the template. Default=0",
-                      mandatory=False)
+    optional.add_argument(
+        "-x",
+        help="Final interpolation.",
+        choices = ("nn", "linear", "spline"),
```

Comment:
as mentioned before, no space around "=" (PEP8)

Reply:
Oh my mistake. Changed in commit [d151dab](https://github.com/neuropoly/spinalcordtoolbox/commit/d151daba8828921ce84006da053bfbfb6694016c)